Proposal: Harold Hearst, Demi Karavoussianis, Kyle Reese, Thomas White

thearst3rd opened the pull request:
We wrote this a while ago but did not submit a pull request, oops....

dkaravoussianis committed:
Create proposal.md

Commit:
Initial commit - setup nodejs+express app

thearst3rd committed:
Added package.lock file, per request of npm

tmwbook committed:
DevOps

Fix glitch 'bundled' packages not being present

tmwbook committed:
Frontend

Add a react app toolchain

React dev server will run on 3333

charlieroberts commented:
We spoke in class, this mostly looks good! Please flush out the details a bit... what database you're going to use, how you're going to do authentication etc. Thanks!

tmwbook committed:
Merge pull request #1 from dkaravoussianis/add-react

Frontend

kdreese committed:
Added database

thearst3rd committed:
Implemented authentication (#3)

* Added passport authentication+pwd hashing and salting, and login and signup endpoints

* Fixed many bugs including adding body-parser middleware. Almost fully authenticates, except that it errors on successful login

* Fixed serialization

* Validate that fields are filled on account signup, and made console.logs more descriptive

kdreese committed:
Removed unneccessary carryover from copy source

tmwbook committed:
Create Database (#2)

* Added database

* Removed unneccessary carryover from copy source

dkaravoussianis committed:
functional login

kdreese committed:
Added helper files

kdreese committed:
Merge branch 'master' into add-database

tmwbook committed:
Card Wireframe

First crack at a tech wireframe for the 'simple' view.

Uses the z-index, with absolute positioning, to stack elements.

kdreese committed:
Added export for functions and streamlined access to database

tmwbook committed:
Merge pull request #4 from dkaravoussianis/add-database

Added helper functions to database

thearst3rd committed:
Added endpoints related to user tasks

thearst3rd committed:
User database is now functional!

thearst3rd committed:
bug fix - switching computers

thearst3rd committed:
Changed some db functions - backend should be fully implemented (unless we want to delete tasks)

tmwbook committed:
Raise state

dkaravoussianis committed:
all this work

tmwbook committed:
Merge feature code

tmwbook committed:
Fix package lock, remove undefined symbol

tmwbook committed:
Merge pull request #6 from dkaravoussianis/card-experiment

Card experiment

kdreese committed:
Updated README

kdreese committed:
Update README.md

tmwbook committed:
Implement card stacking

tmwbook committed:
Merge pull request #5 from dkaravoussianis/hth-tasks

Integrated database into the backend. Implemented task-related endpoints

tmwbook committed:
Replace hardcoded card text with props

tmwbook committed:
Replace hardcoded card text with props

tmwbook committed:
Merge branch 'jerry-rig' of https://github.com/dkaravoussianis/fp into jerry-rig

tmwbook committed:
Merge pull request #7 from dkaravoussianis/jerry-rig

Jerry rig

thearst3rd committed:
Fixed typos

dkaravoussianis committed:
Update README.md

dkaravoussianis committed:
Update README.md

tmwbook committed:
Added animation example

tmwbook committed:
Merge pull request #8 from dkaravoussianis/card-animation

Added animation example

dkaravoussianis committed:
more stuff

dkaravoussianis committed:
Merge remote-tracking branch 'origin/master' into demi

thearst3rd committed:
Added delete functionality

thearst3rd committed:
typo

thearst3rd committed:
Added promise return statements

thearst3rd committed:
Merge pull request #9 from dkaravoussianis/hth-sort

Added deletion functionality

dkaravoussianis committed:
New Task Button

thearst3rd committed:
Sorting algorithm implemented

tmwbook committed:
Render everything

tmwbook committed:
Merge remote-tracking branch 'origin/master' into hoooo-boy

tmwbook committed:
Merge

tmwbook committed:
Add hidden back

thearst3rd committed:
fix typo

thearst3rd committed:
Backend should???? be done

thearst3rd committed:
Merge pull request #10 from dkaravoussianis/hth-sort

Implement sorting algorithm

tmwbook committed:
Big push

tmwbook committed:
Promise you wont get mad

tmwbook committed:
Merge remote-tracking branch 'origin/master' into hoooo-boy

tmwbook committed:
big push (it works)

except for the priority

tmwbook committed:
Fixed priority not getting set

thearst3rd committed:
Update README.md

thearst3rd committed:
Merge pull request #11 from dkaravoussianis/hth-readme

Update README.md

tmwbook committed:
Add to README

tmwbook committed:
Merge pull request #12 from dkaravoussianis/hoooo-boy

Supa Hot

thearst3rd committed:
Added submission link

thearst3rd committed:
Updated readme

thearst3rd committed:
Merge pull request #13 from dkaravoussianis/hth-submission

Added submission link

thearst3rd committed:
Update README.md

thearst3rd committed:
Update README.md again...

thearst3rd committed:
Merge pull request #14 from dkaravoussianis/submission

Submission
